src/map/mob.cpp

```diff
@@ -2417,8 +2417,7 @@ void mob_log_damage(struct mob_data *md, struct block_list *src, int damage)
 void mob_damage(struct mob_data *md, struct block_list *src, int damage)
 {
 	if( src != nullptr && md->special_state.ai == AI_SPHERE && !md->dmglog[0].id ) {//LOne WOlf explained that ANYONE can trigger the marine countdown skill. [Skotlex]
-		md->state.alchemist = 1;
-		mobskill_use(md, gettick(), MSC_ALCHEMIST);
```

Comment:
If we remove this call, then MSC_ALCHEMIST is never used at all.

Reply:
It's a slight trick. It is used in the switch case where because it checks `ms[i]->cond` since NPC_RUN uses alchemist as the condition for Marine Sphere. We want to call NPC_RUN only when the value of can_escape is 1. Before we were making the MSC_ALCHEMIST call before the mob's state was changed/saved resulting in the flag=1 and the skill firing off too early.